File: src/main/java/com/example/common/entity/SchemeRating.java
package com.example.common.entity;

import javax.persistence.*;
import java.util.List;

/**
 * Created by Anatol on 12.06.16.
 */
@Entity
@Table(name = "scheme_rating",uniqueConstraints = @UniqueConstraint(columnNames={"userId", "schemeId"}))
public class SchemeRating extends BaseEntity {
    @Column(nullable = false)
    private Integer value;

    @Column(nullable = false)
    private Long userId;

    @Column(nullable = false)
    private Long schemeId;

    public Integer getValue() {
        return value;
    }

    public void setValue(Integer value) {
        this.value = value;
    }

    public Long getUserId() {
        return userId;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }

    public Long getSchemeId() {
        return schemeId;
    }

    public void setSchemeId(Long schemeId) {
        this.schemeId = schemeId;
    }
}

File: frontend/src/app/dto/SchemeRating.ts
export default class SchemeRating {
    private id: number;
    private value: number
    private userId: number;
    private schemeId: number;

    constructor(id:number, value:number, userId:number, schemeId:number) {
        this.id = id;
        this.value = value;
        this.userId = userId;
        this.schemeId = schemeId;
    }

   public getId():number {
        return this.id;
    }
   
    public setIdd(value:number) {
        this.id = value;
    }

    public  getValue():number {
        return this.value;
    }

    public  setValue(value:number) {
        this.value = value;
    }
    
    public getUserId():number {
        return this.userId;
    }
    
    public setUserId(value:number) {
        this.userId = value;
    }
    
    public getSchemeId():number {
        return this.schemeId;
    }
    
    public setSchemeId(value:number) {
        this.schemeId = value;
    }
}